src/lib/strings/time.ts

```diff
@@ -1,12 +1,36 @@
+import {DateTimeFormat} from '@formatjs/intl-datetimeformat'
```

Comment:
We'd like to avoid importing something that isn't used on the web. Let's instead fork this file like `time.ts` vs `time.web.ts` and keep this import in the native version. This also lets you remove the inline checks.

Reply:
In fact, maybe it's worth moving the code into `time.android.ts` if we're sure the fix is only needed there